codalab/lib/bundle_cli.py

```diff
@@ -1054,6 +1055,14 @@ def do_store_command(self, args):
                 "url": args.url,
                 "authentication": args.authentication,
             }
+            if args.url is not None:
+                inferred_type = parse_linked_bundle_url(args.url).storage_type
+                if args.storage_type is None:
+                    bundle_store_info["storage_type"] = inferred_type
```

Comment:
Is bundle_store_info["storage_type"] set when args.storage_type is specified?

Reply:
I think No? When storage_type is specified, it should have higher priority than inferred_type. And if a user use a wrong command like "--storage_type azure_blob --url gs://bucket", we will raise Error. If the args.storage_type is specified and equals inferred_type, it's ok not to set it.